common/lib/dependabot/file_updaters/base.rb

```diff
@@ -1,18 +1,41 @@
-# typed: true
+# typed: strict
 # frozen_string_literal: true
 
+require "sorbet-runtime"
+
 module Dependabot
   module FileUpdaters
     class Base
-      attr_reader :dependencies, :dependency_files, :repo_contents_path,
-                  :credentials, :options
+      extend T::Sig
+      extend T::Helpers
+      abstract!
+
+      sig { returns(T::Array[Dependabot::Dependency]) }
+      attr_reader :dependencies
+      sig { returns(T::Array[Dependabot::DependencyFile]) }
+      attr_reader :dependency_files
+      sig { returns(T.nilable(String)) }
+      attr_reader :repo_contents_path
+      sig { returns(T::Array[T::Hash[String, String]]) }
+      attr_reader :credentials
+      sig { returns(T::Hash[Symbol, T.untyped]) }
+      attr_reader :options
```

Comment:
I'd find this easier to parse if we added a blank line between each sig+attribute duple. What do you think?

Reply:
Absolutely! Can do.